docs/pl/docs/features.md

```diff
@@ -0,0 +1,200 @@
+
+Dzięki **FastAPI** otrzymujesz wszystkie funkcje **Starlette** (ponieważ FastAPI to po prostu Starlette na sterydach):
+
+* Bardzo imponująca wydajność. Jest to <a href="https://github.com/encode/starlette#performance" class="external-link" target="_blank">jeden z najszybszych dostępnych frameworków Pythona, na równi z **NodeJS** i **Go**</a>.
+* Wsparcie dla **WebSocket**.
+* <abbr title='Zadania wykonywane w tle, bez zatrzymywania żądań, w tym samym procesie. Po angielsku: In-process background tasks'>Zadania w tle</abbr>.
+* Eventy startup i shutdown.
+* Klient testowy zbudowany na bazie biblioteki `requests`.
+* **CORS**, GZip, pliki statyczne, streamy.
+* Obsługa **sesji i ciasteczek**.
+* 100% pokrycie testami.
+* 100% adnotacji typów.
+
+## Cechy Pydantic
+
+**FastAPI** jest w pełni kompatybilny z (oraz bazuje na) <a href="https://pydantic-docs.helpmanual.io" class="external-link" target="_blank"><strong>Pydantic</strong></a>. Tak więc każdy dodatkowy kod Pydantic, który posiadasz, również będzie działał.
+
+Wliczając w to zewnętrzne biblioteki, również oparte o Pydantic, takie jak <abbr title="Mapowanie obiektowo-relacyjne. Po angielsku: Object-Relational Mapper">ORM</abbr>, <abbr title="Object-Document Mapper">ODM</abbr> dla baz danych.
+
+Oznacza to, że w wielu przypadkach możesz przekazać ten sam obiekt, który otrzymasz z żądania **bezpośrednio do bazy danych**, ponieważ wszystko jest walidowane automatycznie.
+
+Działa to również w drugą stronę, w wielu przypadkach możesz po prostu przekazać obiekt otrzymany z bazy danych **bezpośrednio do klienta**.
+
+Dzięki **FastAPI** otrzymujesz wszystkie funkcje **Pydantic** (ponieważ FastAPI bazuje na Pydantic do obsługi wszystkich danych):
+
+* **Bez prania mózgu**:
+    * Brak nowego mikrojęzyka do definiowania schematu, którego trzeba się nauczyć.
+    * Jeśli znasz adnotacje typów Pythona to wiesz jak używać Pydantic.
+* Dobrze współpracuje z Twoim **<abbr title='Skrót od "Integrated Development Environment", podobne do edytora kodu'>IDE</abbr>/<abbr title="Program, który sprawdza Twój kod pod kątem błędów">linterem</abbr>/mózgiem**:
+    * Ponieważ struktury danych Pydantic to po prostu instancje klas, które definiujesz; autouzupełnianie, linting, mypy i twoja intuicja powinny działać poprawnie z Twoimi zweryfikowanymi danymi.
+* **Szybkość**:
+    * w <a href="https://pydantic-docs.helpmanual.io/benchmarks/" class="external-link" target="_blank">benchmarkach</a> Pydantic jest szybszy niż wszystkie inne testowane biblioteki.
+* Walidacja **złożonych struktur**:
+    * Wykorzystanie hierarchicznych modeli Pydantic, Pythonowego modułu `typing` zawierającego `List`, `Dict`, itp.
+    * Walidatory umożliwiają jasne i łatwe definiowanie, sprawdzanie złożonych struktur danych oraz dokumentowanie ich jako JSON Schema.
+    * Możesz mieć głęboko **zagnieżdżone obiekty JSON** i wszystkie je weryfikować i adnotować.
```

Comment:
**weryfikować** i adnotować

once more you used 'weryfikować' instead of 'walidować' which would be more consistent with the rest of your translation.
'Walidacja' is an actual word in polish, though I agree that using it as a verb (walidować) is not something you see often.

Maybe this should be: 'poddać walidacji'
This way it could be both correct and more pleasant to look at. :thinking:

Reply:
Yes, I agree that "poddać walidacji" sounds much better, thanks!